bvm/ballerina-runtime/src/main/java/org/ballerinalang/jvm/TableProvider.java

```diff
@@ -204,7 +203,37 @@ private String generateCreateTableStatment(String tableName, BType constrainedTy
         return sb.toString();
     }
 
-    private String generateCreateTableStatment(String query, String newTableName) {
+    private void generateCreateTableStatement(int type, BArrayType sf, StringBuilder sb) {
```

Comment:
Any special reason to add another method for this? It has introduced two switch cases (One in the caller method and the other in this method)